### PR TITLE
WordPress docs updates

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/cms/wordpress/configuration.md [hunk 1]
```diff
@@ -6,22 +6,35 @@ To configure the *Auth0 for WordPress* plugin, you will need to copy the *Domain
 
 You must first create an app in the Auth0 dashboard before you can configure the *Auth0 for WordPress* plugin. If you already have created the app you want to connect to WordPress, you can skip to the next section.
 
-1. Login to the [Auth0 dashboard](${manage_url}). (If you don't already have an Auth0 account, you can [create one](https://auth0.com).
-2. Navigate to the *Apps / APIs* page and click **+ New App / API**.
-3. In the *Create Application* window, name your app and click **Save**.
+1. Log in to the [Auth0 dashboard](http://${manage_url}). (If you don't already have an Auth0 account, you can [create one](https://auth0.com).
```
Comment (mpaktiti) [inline articles/cms/wordpress/configuration.md]: Hey @khorne3 I think the `http://` should be removed from `(http://${manage_url})`. In the heroku preview at least it is displayed as `http//https://manage.auth0.com`.

File: articles/cms/wordpress/configuration.md [hunk 2]
```diff
@@ -46,7 +59,7 @@ You must first create an app in the Auth0 dashboard before you can configure the
 
 ### Connections
 
-Enable the supported social identity providers you want to allow users to login with. You can configure your own app keys and settings for these connections in the [Auth0 Dashboard](${manage_url}/#/connections/social).
+Enable the supported social identity providers you want to allow users to login with. You can configure your own app keys and settings for these connections in the [Auth0 Dashboard](http://${manage_url}/#/connections/social).
```
Comment (mpaktiti) [inline articles/cms/wordpress/configuration.md]: @khorne3 Same thing here, the default value of `${manage_url}` should be `https://manage.auth0.com` so the `http://` is not needed. The URL at the live doc navigated at `https://manage.auth0.com/#/connections/social`.

File: articles/cms/wordpress/installation.md [hunk 3]
```diff
@@ -1,24 +1,62 @@
...
+
+![](/media/articles/cms/wordpress/plugins.png)
+
+Click **Add New**.
+
+![](/media/articles/cms/wordpress/add-new.png)
+
+Click on **Upload Plugin**.
+
+![](/media/articles/cms/wordpress/upload-and-install.png)
```
Comment (mpaktiti) [inline articles/cms/wordpress/installation.md]: @khorne3 Can you add a couple of words at the images alternate text in this doc?

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey, I left a couple of comments on links and missing alt text, when you have some time take a look. Thanks!

**Comment by khorne3** [comment]
@mpaktiti Fixed. I'm unsure why GitHub doesn't think I've made the requested changes and marked your suggestions as outdated, though.

**Review by mpaktiti — APPROVED** [review]

**Pull request merged**